src/pages/Tap.vue

```diff
@@ -7,7 +7,6 @@
                 <h1>Type a voucher code</h1>
 
                 <form id="textVoucher" v-on:submit.prevent>
-                    <label for="voucherBox" id="lblVoucherBox">Type a voucher code</label>
```

Comment:
Is it bad form to not have a label at all - even an invisible one for accessibility?